relationalai-sdk-snowflake/relationalai/snowflake_sdk/api.py

```diff
@@ -0,0 +1,269 @@
+    return session.sql(f"select LOAD_MODEL('{database}', '{engine}', '{name}', '{path}')").collect()
+
+
+def load_model_code(session: Session, database: str, engine: str, name: str, code: str) -> List[Row]:
+    return session.sql(f"select LOAD_MODEL_CODE('{database}', '{engine}', '{name}', '{code}')").collect()
+
+
+def load_model_query(session: Session, name: str, path: str) -> List[Row]:
+    return session.sql(f"select LOAD_MODEL_QUERY('{name}', '{path}')").collect()
+
+#################################
+# Data Stream
+#################################
+
+
+def create_data_stream(session: Session, data_source: str, database: str, relation: str) -> List[Row]:
+    return session.sql(f"call CREATE_DATA_STREAM('{data_source}', '{database}', '{relation}')").collect()
+
+
+def delete_data_stream(session: Session, data_source: str) -> List[Row]:
+    return session.sql(f"call DELETE_DATA_STREAM('{data_source}')").collect()
+
+
+def get_data_stream(session: Session, data_source_fq_name: str) -> DataFrame:
+    return session.sql(f"""
+        select
+            res:id::string            as id,
+            res:account::string       as account,
+            res:createdBy::string     as created_by,
+            res:createdOn::string     as created_on,
+            res:dbLink::string        as database_link,
+            res:integration::string   as integration,
+            res:name::string          as name,
+            res:state::string         as state,
+            res:rai::object           as rai,
+            res:snowflake::object     as snowflake
+        from
+            (select GET_DATA_STREAM('{data_source_fq_name}') as res)
+    """)
+
+
+def get_data_stream_status(session: Session, data_source_name: str) -> DataFrame:
+    return session.sql(f"call GET_DATA_STREAM_STATUS('{data_source_name}')")
+
+
+def list_data_streams(session: Session) -> DataFrame:
+    return session.sql(f"""
+        select
+            value:id::string            as id,
+            value:account::string       as account,
+            value:createdBy::string     as created_by,
+            value:createdOn::string     as created_on,
+            value:dbLink::string        as database_link,
+            value:integration::string   as integration,
+            value:name::string          as name,
+            value:state::string         as state,
+            value:rai::object           as rai,
+            value:snowflake::object     as snowflake
+        from
+            (select LIST_DATA_STREAMS() as res),
+            lateral flatten (input => res)
+    """)
+
+#################################
+# Misc
+#################################
+
+
+def load_data(session: Session, database: str, relation: str, primary_key: str, query: str) -> List[Row]:
+    return session.sql(f"select LOAD_DATA('{database}', '{relation}', '{primary_key}', '{query}')").collect()
+
+
+def ping(session: Session) -> List[Row]:
```

Comment:
for later iterations: the combination of use_schema (pointing to an integration) and ping (making sure the integration works) could be a good session setup function